legs:
  joint_states_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

  FL_hip_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: FL_hip_joint
  FL_thigh_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: FL_thigh_joint
  FL_calf_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: FL_calf_joint
  FR_hip_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: FR_hip_joint
  FR_thigh_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: FR_thigh_joint
  FR_calf_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: FR_calf_joint
  RL_hip_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: RL_hip_joint
  RL_thigh_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: RL_thigh_joint
  RL_calf_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: RL_calf_joint
  RR_hip_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: RR_hip_joint
  RR_thigh_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: RR_thigh_joint
  RR_calf_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: RR_calf_joint